book/getting-started/run-a-wallaroo-application-docker.md

```diff
@@ -19,13 +19,25 @@ Let's get started!
 
 ## Terminal 1, Start the Wallaroo Docker container
 
-```bash
+{% codetabs name="UNIX Bash", type="bash" -%}
 docker run --rm -it --privileged -p 4000:4000 \
 -v /tmp/wallaroo-docker/wallaroo-src:/src/wallaroo \
 -v /tmp/wallaroo-docker/python-virtualenv:/src/python-virtualenv \
 --name wally \
 wallaroo-labs-docker-wallaroolabs.bintray.io/{{ docker_version_url }}
-```
+{%- language name="Windows Powershell", type="bash" -%}
+docker run --rm -it --privileged -p 4000:4000 |
+-v /tmp/wallaroo-docker/wallaroo-src:/src/wallaroo |
+-v /tmp/wallaroo-docker/python-virtualenv:/src/python-virtualenv |
+--name wally |
```

Comment:
On second thought, the virtual env probably can't be mapped. The execute flag won't be preserved in this case (it would be via WSL but we're using docker.exe instead of linux docker over a unix socket). I can add a note on how to use WSL (recommended when using docker to avoid the flag issues above since the bash example "just works").

Reply:
Also keep in mind the permissions on the wallaroo directory will also have issues. I'm wondering if we should show how to setup WSL docker to connect to docker for windows?

Reply:
@strmpnk what version requires the `--platform=linux`? I tested with 18.03.1 and it wasn't required

Reply:
It is required anytime windows containers are enabled (platform=windows becomes the default), which is default in the current beta release (18.05.0-cd-rc1, and about 2 months prior to this version so this isn't new) and can't easily be changed. Docker is not clear on when that will roll out to stable but I'd expect it sometime this summer.